control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -1650,6 +1677,81 @@ func TestRun_AlreadyBootstrapped(t *testing.T) {
 	}, consulAPICalls)
 }
 
+// Test if there is an old bootstrap Secret and the server token exists
```

Comment:
I added this to test that we don't re-create the server tokens if the token already exists.